Core/Config.cpp

```diff
@@ -875,7 +875,7 @@ static ConfigSetting graphicsSettings[] = {
 	ConfigSetting("CameraDevice", &g_Config.sCameraDevice, "", true, false),
 	ConfigSetting("VendorBugChecksEnabled", &g_Config.bVendorBugChecksEnabled, true, false, false),
 	ConfigSetting("UseGeometryShader", &g_Config.bUseGeometryShader, true, true, true),
-	ReportedConfigSetting("RenderingMode", &g_Config.iRenderingMode, 1, true, true),
+	ConfigSetting("SkipBufferEffects", &g_Config.bSkipBufferEffects, false, true, true),
```

Comment:
Hm, this setting can still affect whether it's running well, so maybe should still be reported.  It can factor into the recommendations it gives you if you say the game isn't running well, i.e. turn off skip buffer effects.

-[Unknown]

Reply:
oops, yeah.